spring-data-relational/src/main/java/org/springframework/data/relational/core/sql/SqlIdentifier.java

```diff
@@ -74,24 +75,13 @@ public String toString() {
 	};
 
 	/**
-	 * Return the reference name after applying {@link IdentifierProcessing} rules. The reference name is used for
-	 * programmatic access to the object identified by this {@link SqlIdentifier}.
-	 *
-	 * @param processing identifier processing rules.
-	 * @return
-	 */
-	String getReference(IdentifierProcessing processing);
-
-	/**
-	 * Return the reference name without any further transformation. The reference name is used for programmatic access to
+	 * Return the name without any further transformation. The reference name is used for programmatic access to
```

Comment:
We shouldn't directly remove public methods. Instead please deprecate it and all it's implementations. We then should remove it with 3.0

Reply:
Fixed with [commit](https://github.com/spring-projects/spring-data-jdbc/pull/1111/files#diff-ad9a19ceaf2639443aa86e26bc0cd97e5e9b2aad5ea158cae9477246b5d38294L82)